packages/pytest-simcore/src/pytest_simcore/logging_utils.py

```diff
@@ -80,17 +102,35 @@ def log_context(
     else:
         ctx_msg = msg
 
+    started_time = datetime.datetime.now(tz=datetime.timezone.utc)
     try:
         DynamicIndentFormatter.increase_indent()
 
         logger.log(level, ctx_msg.starting, *args, **kwargs)
 
         yield SimpleNamespace(logger=logger, messages=ctx_msg)
-
-        logger.log(level, ctx_msg.done, *args, **kwargs)
+        elapsed_time = datetime.datetime.now(tz=datetime.timezone.utc) - started_time
+        done_message = (
+            f"{ctx_msg.done} ({_timedelta_as_minute_second_ms(elapsed_time)})"
+        )
+        logger.log(
+            level,
+            done_message,
+            *args,
+            **kwargs,
+        )
 
     except:
-        logger.log(logging.ERROR, ctx_msg.raised, *args, **kwargs)
+        elapsed_time = datetime.datetime.now(tz=datetime.timezone.utc) - started_time
```

Comment:
I would add this as an `extra` and separate the formatting instead of compose the message.  similar to the way we did with the indentation (see ` DynamicIndentFormatter`).

Later on, the log caller might decide to show or not the elapsed time.  The approach you follow imposes the message while the alternative I suggest makes it optional.

Reply:
Actually wondering whether having it disabled make sense? Why should it be optional? We want people reading these logs to understand right away what is going on right. What is the point of disabling it?